lms/djangoapps/django_comment_client/tests/test_utils.py

```diff
@@ -178,6 +178,22 @@ def assertThreadCorrect(thread, discussion, expected_title):  # pylint: disable=
         assertThreadCorrect(threads[0], self.discussion1, "Chapter / Discussion 1")
         assertThreadCorrect(threads[1], self.discussion2, "Subsection / Discussion 2")
 
+    def test_if_discussion_subcatergory_is_empty(self):
+        """
+        Test that for empty subcategory inline discussion modules,
+        the divider " / " is not rendered on a post or inline discussion topic label.
+        """
+        discussion = ItemFactory.create(
+            parent_location=self.course.location,
+            category="discussion",
+            discussion_id="discussion",
+            discussion_category="Chapter",
+            discussion_target=""  # discussion-subcategory
+        )
+        thread = {"commentable_id": discussion.discussion_id}
+        utils.add_courseware_context([thread], self.course, self.user)
+        self.assertNotEqual(thread.get("courseware_title"), 'Chapter / ')
```

Comment:
I would simply assert that it doesn't contain a "/" to make it a bit more resilient.